### PR TITLE
BETA: Register johndoe.is-a.dev

### PR DESCRIPTION
**Pull request by Jameseson11**
Added `johndoe.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/idontknow.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Jameseson11",
+        "email": "622353@lkstevens.wednet.edu"
+    },
+    "record": {
+        "TXT": "https://mirror.touhidur.tech/"
```
Comment (Stef-00012) [inline domains/idontknow.json:7]: URL records alone do nothing (maybe you meant URL?)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: !@#$%^&*()QSXEDCRFVTGBYHN.is-a.dev
Comment (Stef-00012) [inline !@#$%^&*()QSXEDCRFVTGBYHN.is-a.dev:1]: You can't add files to the root of the repo

File: domains/thth.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Jameseson11",
+        "email": "622353@lkstevens.wednet.edu"
+    },
+    "record": {
+        "CNAME": "thth.pages.dev"
```
Comment (Stef-00012) [inline domains/thth.json:7]: I cannot reach your site
"This site can’t be reached"

File: domains/ui.pages.dev.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Jameseson11",
+        "email": "622353@lkstevens.wednet.edu"
+    },
+    "record": {
+        "CNAME": "ui.pages.dev"
```
Comment (Stef-00012) [inline domains/ui.pages.dev.json:7]: I cannot reach your website (404)

File: domains/uio.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Jameseson11",
+        "email": "622353@lkstevens.wednet.edu"
+    },
+    "record": {
+        "CNAME": "avnoor-gamerz.github.io"
```
Comment (Stef-00012) [inline domains/uio.json:7]: I cannot reach your website (404)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**